container_images/daisy-builder/main.sh

```diff
@@ -20,6 +20,17 @@ ZONE="$2"
 WORKFLOW_FILE="$3"      # Workflow to run
 GCS_OUTPUT_BUCKET="$4"  # Destination for artifacts
 
+function generate_new_version() {
+  local VERSION=`/versiongenerator --token-file-path=${GITHUB_ACCESS_TOKEN} --org=${REPO_OWNER} --repo=${REPO_NAME}`
+
+  if [[ $? -ne 0 ]]; then
+    echo -e "could not generate version: $VERSION_GENERATE_OUT"
```

Comment:
i don't think `$VERSION_GENERATE_OUT` is set

Reply:
done